db/seeds/warning_templates/spam.md

```diff
@@ -0,0 +1,7 @@
+We've noticed that in addition to your contributions to $SiteName, you've also been using this account to promote a product, service, or similar.
+
+This is a gentle reminder that using an account for promotional purposes is against the rules of this site.
+
+Please take a moment to review the guidelines for promotional content here on $SiteName, which can be found in the [help center](/policy/spam).

```

Comment:
Is the spam topic under development separately?  I don't see it on this PR, though I saw the draft in chat so I know it's floating around somewhere.

Reply:
The spam mod message is in here, but the spam help topic is not - that'll go in separately when it's ready

Reply:
I'll add it (too in this PR, 'cause it makes sense), I just haven't had time yet.